### ISSUE TITLE
InfluxDB Enterprise 1.9.7 release notes 

### ISSUE THREAD

**Issue by kelseiv** [influxdata/docs-v2#3922]
https://dl.influxdata.com/enterprise/nightlies/master/CHANGELOG.md

cc @samhld

**Comment by samhld** [comment]
@kelseiv This is a dupe of https://github.com/influxdata/docs-v2/issues/3928, right?

**Comment by samhld** [comment]
Closes https://github.com/influxdata/docs-v2/issues/3928

## Features
- Expose passive nodes feature to `influxd-ctl` and the API. Notes for docs: https://github.com/influxdata/docs-v2/issues/3794
- Throttle inter-node data replication, both incoming writes and hinted hand-off, when errors are encountered (for docs, this pertains to https://github.com/influxdata/plutonium/issues/3790)
- Update Flux to 0.161.0
  - Add http/requests package!
  - Add isType()
  - Add display()
  - Function enhancements: `increase()`, `sort()`, `derivative()`, `union()`, `timeshift()`, vectorization to applicable functions (i.e. `map()`)
  - Continue performance work (i.e., sort-> limit)
  - Add TCP connection pooling to `mqtt.publish()` when called in `map()`


## Maintenance updates
- Update Go to 1.17.9
- Update Flux to [0.161.0](https://docs.influxdata.com/flux/v0.x/release-notes/#v01590-2022-03-14)

## Bug fixes
Error messaging
- Improve error messaging for `max series per database exceeded` error
- Add info to error creating subscription

Miscellaneous
- Fix race condition causing restore command to fail

@kelseiv @lwandzura for review

**Comment by samhld** [comment]
@davidby-influx can you review the above comment for any omissions, to go out for release notes?

**Comment by davidby-influx** [comment]
We are also going get fixes for 
https://github.com/influxdata/EAR/issues/3193 (gating fix for 1.9.7 GA)
https://github.com/influxdata/plutonium/pull/3805

**Comment by samhld** [comment]
Thanks, @davidby-influx  -- added!

**Comment by lwandzura** [comment]
@samhld wasn't "Drop shard should ignore shard not found errors" fixed in 1.9.6?  Is this an additional update?

![image](https://user-images.githubusercontent.com/51929958/165789833-2054a46c-331c-4906-a047-3413835a36e6.png)

**Comment by samhld** [comment]
@lwandzura that's very weird. It looks like that incorrectly went into 1.9.6. Any idea how that's even possible? Did 1.9.6 tell the future? That commit is not in the commit log for that version.

**Comment by lwandzura** [comment]
it is weird....but miscommunications do happen sometimes

**Comment by samhld** [comment]
@lwandzura of course! I'm actually just curious how 1.9.6 was able to tell the future :)

**Comment by lwandzura** [comment]
@samhld can you add any detail around this bullet point?  do you mean we are continuing to improve flux functions in general or how they work within Enterprise?

Continue performance work (i.e., sort-> limit)

**Comment by samhld** [comment]
@lwandzura 

> do you mean we are continuing to improve flux functions in general or how they work within Enterprise?

It's both. Flux gets upgrades in each of these versions and that often comes with features or optimizations. Flux function improvements are not only that but also improvements to Enterprise...because it's part of it.
